android/app/src/main/java/app/organicmaps/Framework.java

```diff
@@ -187,11 +187,11 @@ public static void setSpeedCamerasMode(@NonNull SettingsPrefsFragment.SpeedCamer
 
   public static native DistanceAndAzimut nativeGetDistanceAndAzimuthFromLatLon(double dstLat, double dstLon, double srcLat, double srcLon, double north);
 
-  public static native String nativeFormatLatLon(double lat, double lon, int coordFormat);
+  public static native String nativeFormatLatLon(double lat, double lon, int coordFormat); // TODO: move this method to StringUtils class.
 
-  public static native String nativeFormatAltitude(double alt);
+  public static native String nativeFormatAltitude(double alt); // TODO: move this method to StringUtils class.
 
-  public static native String nativeFormatSpeed(double speed);
+  public static native String nativeFormatSpeed(double speed); // TODO: move this method to StringUtils class.
```

Comment:
I left these `TODO` comments to make changes later. Too many changes for a single PR.